ts4.1/icu.d.ts

```diff
@@ -0,0 +1,85 @@
+declare module 'react-i18next/icu.macro' {
+  type Namespace = import('.').Namespace;
+  type DefaultNamespace = import('.').DefaultNamespace;
+  type Resources = import('.').Resources;
+  type TFuncKey<N extends Namespace = DefaultNamespace, T = Resources> = import('.').TFuncKey<N, T>;
+  type i18n = import('i18next').i18n;
+
+  export interface PluralSubProps<
```

Comment:
As I mentioned above all types inside the module will be exported so we don't need the `export` keyword